### ISSUE TITLE
Build failure:  firefox-[beta/devedition]-unwrapped-130.0b2

### ISSUE THREAD

**Issue by khaneliman** [NixOS/nixpkgs#339824]
### Steps To Reproduce

1. build firefox-[beta/devedition]-unwrapped-130.0b2

### Build log

https://gist.github.com/khaneliman/622edcdf5cb9baa3ceb1fd3b989eb39d

### Notify maintainers
@jopejoe1

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
🔨 Welcome to khanelinix
 - system: `"x86_64-linux"`
 - host os: `Linux 6.10.7, NixOS, 24.11 (Vicuna), 24.11.20240831.12228ff`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.4`
 - channels(root): `""`
 - nixpkgs: `/nix/store/bd4fmzws6n5542khxbifbkr6nrygi232-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by tinou98** [comment]
It seems that the relevant error from the logs is :
```
thread 'main' panicked at src/bindgen/config.rs:1125:34:
called `Result::unwrap()` on an `Err` value: 
Couldn't parse config file: TOML parse error at line 364, column 1
    |
364 | \"Keyframe\" = \"Keyframe\"
    | ^
      duplicate key `Keyframe` in table `export.rename`.
```

The source file (`servo/ports/geckolib/cbindgen.toml`) is erroneous on 130.0b2, but also in b3 and b4. It seems to be fixed since 130.0b5

**Comment by jopejoe1** [comment]
Should be fixed by #339997
